apps/web/components/dispute/DisputeList.tsx

```
@@ -0,0 +1,112 @@
+                <div style={{ color: '#888', fontSize: 18 }}>No disputes yet.</div>
+            ) : (
+                <table style={{ width: '100%', borderCollapse: 'separate', borderSpacing: 0, background: '#fff', borderRadius: 16, boxShadow: '0 2px 16px #eaeaf4', overflow: 'hidden' }}>
+                    <thead style={{ background: '#f7fafd' }}>
+                        <tr>
+                            <th style={{ padding: 12, textAlign: 'left', fontWeight: 700 }}>ID</th>
+                            <th style={{ padding: 12, textAlign: 'left' }}>Escrow</th>
+                            <th style={{ padding: 12, textAlign: 'left' }}>Milestone</th>
+                            <th style={{ padding: 12, textAlign: 'left' }}>Mediator</th>
+                            <th style={{ padding: 12, textAlign: 'left' }}>Status</th>
+                            <th style={{ padding: 12, textAlign: 'left' }}>Notes</th>
+                            <th style={{ padding: 12, textAlign: 'left' }}></th>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)